README.md

```diff
@@ -0,0 +1,5 @@
+[bsc#999203](https://bugzilla.suse.com/show_bug.cgi?id=999203)
+
+```console
+$ ruby print_deps.rb YaST:Head openSUSE_Factory x86_64 yast_deps.yaml
+```
```

Comment:
OK, you open pandora box. If you would like to add README, then it should at least contain, what is this project good for, its features and describe terms used there ( like definition of critical path ).

ftw for example I think it is more useful to show it for openSUSE:Factory, as when I plan to introduce it, I would like to show it for factory.